middleware/logger.go

```diff
@@ -34,7 +64,7 @@ func Logger() echo.MiddlewareFunc {
 				code = color.Cyan(n)
 			}
 
-			log.Printf("%s %s %s %s %d", method, path, code, stop.Sub(start), size)
+			log.Printf("%s %s %s %s %s %d", parseIP(remoteAddr), method, path, code, stop.Sub(start), size)
```

Comment:
I am okay with the proxy headers, that's how it should be, I was more specific about calling `net.ParseIP` do we really need it, it's just a logger? We should just get the IP, split and display. What do you guys think?